monkeytype/cli.py

```diff
@@ -65,6 +65,10 @@ def complain_about_no_traces(args: argparse.Namespace, stderr: IO) -> None:
     module, qualname = args.module_path
     if qualname:
         print(f'No traces found for specifier {module}:{qualname}', file=stderr)
+    # When there is no trace and a top level module's filename is passed, print
+    # a useful error message.
+    elif os.path.exists(module):
+        print(f'{module} does not look like a valid Python import path', file=stderr)
```

Comment:
Let's tweak this error message a bit; in fact `foo.py` does "look like a valid Python import path", it just happens to likely not be one in this case. Suggested error message: `f"No traces found for {module}; did you pass a filename instead of a module name? Maybe try just '{os.path.splitext(module)[0]}'."`